app/blog/(posts)/1000-days-of-running/page.mdx

```diff
@@ -0,0 +1,173 @@
+  Track workouts work because they let you push as hard as you can, which increases your max speed.
+</Callout>
+
+By the end of the second year, my shorter distances had basically caught up.
+
+### 1000 days (100%) + future
+
+<Callout intent="none">
+  Trained to run faster, shorter distances.
+</Callout>
+
+I&apos;m now comfortable with every type of training (interval training, long runs, lactate threshold, easy runs, etc.), so I decide what to run each day based on what my goals are (while maintaining a base weekly mileage of <Distance m={144_841} />).
+Currently, I&apos;m focusing on the <Distance m={5_000} /> distance because it&apos;s what I find fun right now, I want to build up more speed while I&apos;m still young, and I&apos;m kinda close to breaking a huge barrier (14:59, PB is a 15:49).
+
+The long-term goal is still to get good at running the half marathon/marathon distances&mdash;it would be really cool to qualify for the US Olympic Trials (requires a ~<Pace s={196} /> marathon as of 2025), but I&apos;m not really seriously thinking about it yet because it&apos;s still a long way off.
+
+## Learnings
+
+Beyond the stuff I obviously learned directly about running (e.g. how to run, recover, etc.), here&apos;s a list of things I learned that I thought were interesting:
+- People seem to think I am more disciplined and have better work ethic after finding out I run every day.
+- Running discipline doesn&apos;t translate at all to other aspects of life.
+- Most people don&apos;t even try when they say they run, and they get way too swamped with preconceptions on what they _should_ learn, self-reflect upon, or gain from running, but you can kind of tell they don&apos;t actually feel it.
+In my opinion, it ends up feeling performative and makes the training worse because you don&apos;t learn to properly enjoy and get a feel for running.
+- Long-distance running is very easily impressive to people because most people have tried or run at some point and struggled to catch their breath (even if it&apos;s running just <Distance m={10} /> to catch a traffic light), so they automatically think running far is hard.
+It&apos;s because running is an activity that gets significantly easier with consistency, and most people never push past the initial barrier;
+it&apos;s not nearly as hard as people think.
+This realization formed this mental model of activites that are often much easier than the perception in my head.
+Things like: chess players being geniuses, or Rubik&apos;s cube solvers being geniuses (maybe not anymore).
+- The largest personal growth from running has been it serving as a really strong example of "you can just do things" because I went into the 1000 days with the plan "run <Distance m={128_748} />/week to <Distance m={160_934} />/week for like 10 years, and you can&apos;t be bad," and it seems to be true.
```

Comment:
"because I went into the 1000 days with the plan "run <Distance m={128_748} />/week to <Distance m={160_934} />/week for like 10 years, and you can&apos;t be bad," and it seems to be true."

->

"- I went into the 1000 days with the plan "run <Distance m={128_748} />/week to <Distance m={160_934} />/week for like 10 years, and you can&apos;t be bad," and it seems to be true so far."